build.sbt

```diff
@@ -34,35 +35,13 @@ ThisBuild / printScalaVersion := {
   log.info(s"Local maven ${Resolver.mavenLocal}")
 }
 
-lazy val jacocoCoreReportSettings = Seq(
-  jacocoReportSettings := JacocoReportSettings(
-    s"fa-db:core Jacoco Report",
-    None,
-    JacocoThresholds(),
-    Seq(JacocoReportFormats.HTML, JacocoReportFormats.XML),
-    "utf-8"),
-  jacocoExcludes := Seq(
-//        "za.co.absa.fadb.naming_conventions.SnakeCaseNaming*", // class and related objects
-//        "za.co.absa.fadb.naming_conventions.AsIsNaming" // class only
-  )
-)
-
-lazy val jacocoSlickReportSettings = Seq(
-  jacocoReportSettings := JacocoReportSettings(
-    s"fa-db:slick Jacoco Report",
-    None,
-    JacocoThresholds(),
-    Seq(JacocoReportFormats.HTML, JacocoReportFormats.XML),
-    "utf-8")
+lazy val commonJacocoReportSettings: JacocoReportSettings = JacocoReportSettings(
+  formats = Seq(JacocoReportFormats.HTML, JacocoReportFormats.XML)
```

Comment:
Do I guess correctly that the other settings, like `JacocoThresholds()` that were omitted are default?

Reply:
Yes. I have removed default constructor parameters.